app/javascript/modules/listingDetails/ListingDetailsImageCard.tsx

```diff
@@ -71,7 +71,7 @@ export const ListingDetailsImageCard = ({ listing }: ListingDetailsImageCardProp
   const { fallbackUsed, props: imageCardProps } = createImageCardProps(listing)
 
   return (
-    <header className="image-card--leader">
+    <header className="image-card--leader translate">
```

Comment:
Does adding this `translate` class force the machine translation of the tag content?

Reply:
good catch, yes. I can try applying the `translate` class to only the `figure class="image_card__inner" element (in Bloom UIC). I'll research how to do that, unless you know a good way to apply that class without modifying any Bloom UIC components?